src/devices/cpu/s1c33/s1c33dasm.h

```diff
@@ -0,0 +1,50 @@
+	offs_t handle_010x_xxxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_011x_xxxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_100x_xxxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_1000_00xx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_1000_01xx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_100y_yyxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_100y_00xx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_100y_01xx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_100y_10xx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_100y_11xx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_101x_xxxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_101x_xx00_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_101x_xx01_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_101x_xx10_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_101x_xx11_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_110x_xxxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
+	offs_t handle_111x_xxxx_xxxx_xxxx(std::ostream &stream, offs_t pc, const s1c33_disassembler::data_buffer &opcodes, const s1c33_disassembler::data_buffer &params, u16 op);
```

Comment:
You don’t need `s1c33_disassembler::` inside the `s1c33_disassembler` class declaration when there are no base classes that depend on template arguments (the class isn’t even a template).